src/applications/disability-benefits/new-686/config/constants.js

```diff
@@ -3,6 +3,8 @@ export const numberAndDashPattern = '^[0-9]*[-]*[0-9]*[-]*[0-9]*$';
 export const datePattern =
   '^(\\d{4}|XXXX)-(0[1-9]|1[0-2]|XX)-(0[1-9]|[1-2][0-9]|3[0-1]|XX)$';
 
+export const phonePattern = '^[0-9]{10}';
```

Comment:
It might be worthwhile to add a '$' to the end of this, just to be safe. Without it, having a string of `1234567890a` will still match the numbers, but inserting the 'a' in there won't necessarily make it invalid. So I _think_ if we had an error message based off a pattern, it wouldn't actually get thrown.

Reply:
added this